Network carrier no longer returned for iOS (#5536)
segmentio/segment-docs: src/connections/sources/catalog/libraries/mobile/ios/ios-faqs.md

## AppClip tracking support

If you are tracking App Clips using iOS or Swift libraries, there is a chance that you may encounter zeros in your device ID. Segment recommends that you set your own device ID in this instance to avoid running into this issue.

## Why am I seeing a value of -- set for the network carrier?

> As from iOS [16.4](https://developer.apple.com/documentation/ios-ipados-release-notes/ios-ipados-16_4-release-notes#Core-Telephony), Apple has deprecated the method to return the network carrier. As such, the iOS library can no longer return a valid value for the network carrier on devices using iOS 16.4 or later. You will likely see `--` set for the `context.network.carrier` field.

Reviewer: ```suggestion
With iOS [16.4](https://developer.apple.com/documentation/ios-ipados-release-notes/ios-ipados-16_4-release-notes#Core-Telephony){:target="_blank"}, Apple deprecated the method to return the network carrier. The iOS library can no longer return a valid value for the network carrier on devices using iOS 16.4 or later. As a result, you will likely see `--` set for the `context.network.carrier` field.
```
